### PR TITLE
KEYCLOAK-10610 Update image for CD7

### PR DESCRIPTION
**Pull request by martin-kanis**
Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Review by iankko — APPROVED** [review]
LGTM

@martin-kanis For now put on hold due that TODO action. If you need to build immediately, just use the latest `jboss-eap-modules` commit hash, update the PR, and remove the hold label

Otherwise LGTM in general. Thanks

**Comment by martin-kanis** [comment]
@iankko Updated to CD17_CR2 and 0.35.1 for cct_modules

File: templates/sso-cd-image-stream.json [hunk 1]
```diff
@@ -107,6 +107,34 @@
...
+                            "supports": "sso:7",
+                            "version": "7",
+                            "openshift.io/display-name": "Red Hat Single Sign-On 7 Continuous Delivery (Tech Preview)"
+                        },
+                        "referencePolicy": {
+                            "type": "Local"
+                        },
+                        "from": {
+                            "kind": "DockerImage",
+                            "name": "registry.redhat.io/redhat-sso-7-tech-preview/sso-cd-openshift:7"
```
Comment (iankko) [inline templates/sso-cd-image-stream.json]: @martin-kanis @hmlnarik This change is OK for now. But if we are going to rebase to RHEL-8 image, it needs to be adjusted later to `redhat-sso-cd-tech-preview/sso-cd-openshift-rhel8:7` form, jFYI.

**Review by iankko — APPROVED** [review]
lgtm. Just added one note, what needs to be updated yet, the templates to work with RHEL-8 variant of the image.

**Comment by hmlnarik** [comment]
@martin-kanis Merging #97 has caused the conflicts, could you please rebase?

**Review by hmlnarik — APPROVED** [review]

**Pull request merged**